astropy/cosmology/io/html.py

```diff
@@ -0,0 +1,129 @@
+import astropy.cosmology.units as cu
+import astropy.units as u
+from astropy.cosmology.connect import readwrite_registry
+from astropy.cosmology.core import Cosmology
+from astropy.cosmology.parameter import Parameter
+from astropy.table import QTable
+
+from .table import from_table, to_table
+
+# format look-up for conversion, {original_name:new_name}
+format_table = {"Om0": "testName", "doesNotExist": "testName2", "Ob0": "testName3"}

```

Comment:
This needs real values (and tests). I'm listing all the current Parameters.

- H0
- Om0
- Ode0
- Tcmb0
- Neff
- m_nu
- Ob0
- w0
- wa
- wz
- wp
- zp

Reply:
Would it be ok for testing if we had a table that looks like this:
```
format_table = {"H0": "H0_t",
                "Om0": "Om0_t",
                "Ode0": "Ode0_t",
                "Tcmb0": "Tcmb0_t",
                "Neff": "Neff_t",
                "m_nu": "m_nu_t",
                "Ob0": "Ob0_t",
                "w0": "w0_t",
                "wa": "wa_t",
                "wz": "wz_t",
                "wp": "wp_t",
                "zp": "zp_t"}
```
Or do both the key and value need to be real units? 

The values ending in _t do not exist, but the keys do, which should remain consistent in the actual Cosmology format.

Reply:
The best would be to do the test on a real table and compare the output to expectations — in this case that the table columns are correctly modified and renamed from the original table generated by ``to_table()``.